generative_models/VAE-GAN-tf/README.md

```diff
@@ -0,0 +1,38 @@
+# vae-gan-tf
+
+
+# Paper
+
+* Title: Autoencoding beyond pixels using a learned similarity metric.
+
+* Authors: Anders Boesen Lindbo Larsen,Søren Kaae Sønderby,Hugo Larochelle,Ole Winther.
+
+* Link: https://arxiv.org/pdf/1512.09300.pdf.
+
+* Tags: Neural Network, Generative Networks, GANs.
+
+* Year: 2016.
+
+
+# Summary
+
```

Comment:
It would be nice to know the architecture of paper, and describe the process of generation